server/controllers/message.controller.js

```diff
@@ -177,43 +162,38 @@ function list(req, res) {
         queryObject.where = { ...queryObject.where, ...whereObject };
     }
 
-    if (req.query.limit) {
+    if (req.query.limit !== undefined) {
```

Comment:
Is this because limit and offset could potentially be valued `null`? Or was the simple existence checking creating bugs?

Reply:
`limit=0` is valid but falsy so the original check wouldn't catch it